docs/integrations/webhooks/sync-data.mdx

```diff
@@ -78,27 +76,13 @@ These steps apply to any Clerk event. To make the setup process easier, it's rec
 
   Incoming webhook events have no auth information. They come from an external source and will not be signed in or signed out. To allow access, the route must be public.
 
-  The following example shows the recommended Middleware configuration for webhook routes.
-
-  ```ts {{ filename: 'middleware.ts' }}
-  import { clerkMiddleware } from '@clerk/nextjs/server'
-
-  // Make sure that the `/api/webhooks(.*)` route is not protected here
-  export default clerkMiddleware()
+  Incoming webhook events contain no authentication information. They come from an external source and won't be signed in or signed out, so the route must be public to allow access.
 
-  export const config = {
-    matcher: [
-      // Skip Next.js internals and all static files, unless found in search params
-      '/((?!_next|[^?]*\\.(?:html?|css|js(?!on)|jpe?g|webp|png|gif|svg|ttf|woff2?|ico|csv|docx?|xlsx?|zip|webmanifest)).*)',
-      // Always run for API routes
-      '/(api|trpc)(.*)',
-    ],
-  }
-  ```
+  To configure your Middleware, refer to the [clerkMiddleware() guide](docs/references/nextjs/clerk-middleware).
```

Comment:
looks like you repeat information here. 
also the sentence ends wtih "so the route must be public to allow access". and then the next sentence goes into configuring your middleware.

there should be some relational information that ties these two concepts together:
"so the route must be public to allow access. If you are using `clerkMiddleware()`, you will need to ensure that the `/api/webhooks(.*)` route is public. For more information on configuring routes with `clerkMiddleware()`, see [the reference](/docs/references/nextjs/clerk-middleware).

also don't forget that clerkMiddleware() always needs to be in code snippets. even if it's in a link